acme/acme/standalone.py

```diff
@@ -83,9 +83,18 @@ def __init__(self, ServerClass, server_address, *remaining_args, **kwargs):
                 new_address = (server_address[0],) + (port,) + server_address[2:]
                 new_args = (new_address,) + remaining_args
                 server = ServerClass(*new_args, **kwargs) # pylint: disable=star-args
+                logger.debug("Successfully bound to %s:%s using %s", new_address[0],
+                    new_address[1], "IPv6" if ip_version else "IPv4")
             except socket.error:
-                logger.debug("Failed to bind to %s:%s using %s", new_address[0],
+                if self.servers:
+                    # Already bound using IPv6.
+                    logger.debug("""Certbot wasn't able to bind to %s:%s using
```

Comment:
We have used the following notation for multiline strings elsewhere in the code, and would prefer it for consistency as well as readability:
```
if self.servers:
    # Already bound using IPv6.
    logger.debug("Certbot wasn't able to bind to %s:%s using %s, this is often " +
                 "expected due to the dual stack nature of IPv6 socket implementations.",
                 new_address[0], new_address[1], "IPv6" if ip_version else "IPv4")
```

My apologies for not pointing that out earlier on.

Reply:
No worries, thanks for the tip! 👍